HTTP_200/urls.py

```diff
@@ -21,6 +21,7 @@
     url(r'^contact/$', Contact.as_view(), name='contact'),
     # url(r'^api/', include('feeds.urls')),
     # url(r'', include('rest_framework.urls', namespace='rest_framework'))
+    url(r'^busers/$', BulkUser.as_view(), name='bulk_users'),
```

Comment:
The urlpattern should be named something human readable and that makes some sense. It could be something like `/students/create` or anything meaningful